invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/CommunityProfileForm.js

```diff
@@ -377,178 +378,200 @@ class CommunityProfileForm extends Component {
-                        fieldPath="metadata.description"
-                        label={
-                          <FieldLabel
-                            htmlFor="metadata.description"
-                            icon="pencil"
-                            label={i18next.t("Short description")}
-                          />
-                        }
-                        fluid
-                      />
-                      <SelectField
-                        search
-                        clearable
-                        fieldPath="metadata.type.id"
-                        label={
-                          <FieldLabel
-                            htmlFor="metadata.type.id"
-                            icon="tag"
-                            label={i18next.t("Type")}
-                          />
-                        }
-                        options={types.map((ct) => {
-                          return {
-                            value: ct.id,
-                            text: ct?.title_l10n ?? ct.id,
-                          };
-                        })}
-                      />
-                      <TextField
-                        fieldPath="metadata.website"
-                        label={
-                          <FieldLabel
-                            htmlFor="metadata.website"
-                            icon="chain"
-                            label={i18next.t("Website")}
-                          />
-                        }
-                        fluid
-                      />
 
-                      <RemoteSelectField
-                        fieldPath="metadata.organizations"
-                        suggestionAPIUrl="/api/affiliations"
-                        suggestionAPIHeaders={{
-                          Accept: "application/json",
-                        }}
-                        placeholder={i18next.t("Search for an organization by name")}
-                        clearable
-                        multiple
-                        initialSuggestions={_get(
-                          community,
-                          "metadata.organizations",
-                          []
-                        )}
-                        serializeSuggestions={(organizations) =>
-                          _map(organizations, (organization) => {
-                            // eslint-disable-next-line no-prototype-builtins
-                            const isKnownOrg = this.knownOrganizations.hasOwnProperty(
-                              organization.name
-                            );
-                            if (!isKnownOrg) {
-                              this.knownOrganizations = {
-                                ...this.knownOrganizations,
-                                [organization.name]: organization.id,
-                              };
-                            }
+                      <Overridable
+                        id="CommunityProfileForm.Description"
```

Comment:
May I suggest a small change in the names, even if other existing `id` do not follow (yet) this convention?
https://inveniordm.docs.cern.ch/develop/best-practices/react/#on-overridable

Reply:
I adjusted the names and hope this matches the convention now